code/game/machinery/autolathe.dm

```diff
@@ -15,10 +15,11 @@
 	var/list/storage_capacity = list(DEFAULT_WALL_MATERIAL = 0, "glass" = 0)
 	var/show_category = "All"
 
-	var/hacked = 0
-	var/disabled = 0
-	var/shocked = 0
-	var/busy = 0
+	var/hacked = FALSE
+	var/disabled = FALSE
+	var/shocked = FALSE
+	var/busy = FALSE
+	var/build_item
```

Comment:
Is there a point to having both a `build_item` and a `making` var? Why not just make `build_item` do all the stuff `making` does.